kubernetes/kind/configs/agent/agent.yaml

```diff
@@ -1,13 +1,19 @@
+## This is required by portainer
 apiVersion: v1
 kind: Namespace
 metadata:
   name: portainer
 ---
 apiVersion: v1
+kind: Namespace
+metadata:
+  name: portainer-agent
+---
```

Comment:
Not sure to understand why you let this `portainer` Namespace definition instead of replacing it ?
`portainer` namespace is already defined in portainer.yaml